migrations/versions/d01c4ceddeb4_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
+
+Revision ID: d01c4ceddeb4
+Revises: df617df60678
+Create Date: 2019-03-26 07:27:34.083130
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'd01c4ceddeb4'
+down_revision = 'df617df60678'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('idx_project_info composite', table_name='project_info')
+    op.create_index('idx_project_info composite', 'project_info', ['locale', 'project_id'], unique=False)
+    op.drop_index('idx_project_info composite', table_name='project_info')
```

Comment:
why is it dropped again right after creation?

Reply:
I'm not entirely sure. That part was autogenerated by alembic. I'll remove the `drop`s. I can't think of any reason they would need to be there.